design/work_item_children.go

```diff
@@ -0,0 +1,23 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("work-item-children", func() {
```

Comment:
I have moved the implementation to `workitem/link` package ( Commit: f44ff7b2fbf2c30c19b945b254c5e1673f83584a ) But kept the design in the same file, is that Ok?